libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
But SPEC say
```
// Maximum number of PIDs. A value < 0 implies "no limit".
Limit int64 `json:"limit"`
```
Please consider base on the PR https://github.com/opencontainers/runc/pull/369